lib/authenforce/enforcers.go

```diff
@@ -2,12 +2,15 @@ package authenforce
 
 import (
 	"context"
+	"sync/atomic"
 
 	"github.com/cuvva/cuvva-public-go/lib/cher"
+	"github.com/wearemojo/mojo-public-go/lib/errgroup"
 	"github.com/wearemojo/mojo-public-go/lib/merr"
 )
 
 type (
+	Enforcers   []Enforcer
 	Enforcer    func(context.Context, any, map[string]any) (bool, error)
 	SubEnforcer func(context.Context, any, map[string]any) error
```

Comment:
Perhaps we could add a little bit of info on what the two enforcer types do, what they're used for